File: diana/src/main/java/ai/fasion/fabs/diana/domain/vo/SkuVO.java
package ai.fasion.fabs.diana.domain.vo;

import java.util.Date;

public class SkuVO {

    /**
     * sku主id
     */
    private String id;

    /**
     * sku
     */
    private String sku;

    /**
     * 名称
     */
    private String name;

    /**
     * 价格
     */
    private Integer price;

    /**
     * sku类型
     */
    private String type;

    /**
     * 点数描述
     */
    private String slogan;

    /**
     * 状态： enable--上架  disable--下架
     */
    private String status;

    /**
     * 点数
     */
    private Integer points;

    /**
     * 有效期
     */
    private Integer expirationPeriod;

    /**
     *创建时间
     */
    private Date createdAt;
}

File: apollo/src/main/java/ai/fasion/fabs/apollo/point/vo/SkuVO.kt
package ai.fasion.fabs.mercury.point.vo

/**
 * sku表
 */
data class SkuVO (
    /**
     * 编号
     */
    var id: String? =null,

    /**
     * sku
     */
    var sku:String? =null,

    /**
     * 名称
     */
    var name: String? =null,

    /**
     * 点数描述
     */
    var slogan: String? =null,

    /**
     * 价格(发布会不可修改)
     */
    var price: Int? =null,

    /**
     * 点数
     */
    var points: Int? =null,

    /**
     * 有效期 以天为单位
     */
    var expirationPeriod: Int? =null
)